networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -11,7 +11,9 @@ The Kubernetes NMState Operator provides a Kubernetes API for performing state-d
 
 [IMPORTANT]
 ====
-Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, {ibm-power-name}, {ibm-z-name}, {ibm-linuxone-name}, VMware vSphere, and OpenStack installations.
+Red{nbsp}Hat supports the Kubernetes NMState Operator in production environments on bare metal, {ibm-power-name}, {ibm-z-name}, {ibm-linuxone-name}, {vmw-first}, and {rh-openstack-first} installations.
```

Comment:
```suggestion
Red{nbsp}Hat supports the Kubernetes NMState Operator in production environments on bare-metal, {ibm-power-name}, {ibm-z-name}, {ibm-linuxone-name}, {vmw-first}, and {rh-openstack-first} installations.
```
IDK do we have a no-hyphen prohibition on bare metal? (I didn't find it if so.) Anywho, "bare-metal installation" would take a hyp'en otherwise.

Reply:
Yeah, the rule seems to not pick up that the noun is much further along in the chain. Vale is treating the bare metal entry as a noun instead of an adjective.

![Screenshot from 2024-09-10 15-08-54](https://github.com/user-attachments/assets/1c1cbe7c-bfca-47de-bb44-d976d2d1073e)